### DISCUSSION TITLE
Modal overlay overflow

### DISCUSSION THREAD

**Discussion by john-george-gewan** [mantinedev/mantine#5098] [Q&A]
### What package has an issue

@mantine/modals

### Describe the bug

Is there any way to avoid scrolling inside the modal body for modals with large content and make the whole modal scrollable inside the overlay.

### What version of @mantine/* packages do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

6.0.10

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by 7iomka** [comment]
I have asked a similar question to the author before.
He refused me with the argument that we left this approach in order not to bother with supporting calculations of scrolling blocking and scrollbar size to prevent jumping

**Comment by rtivital** [comment]
You can customize Modal styles with Styles API. If you need more control over the rendering of the component, use compound components to build a custom modal – https://mantine.dev/core/modal/#compound-components
